packages/create-amplify/templates/basic-auth-data/amplify/auth/resource.ts

```diff
@@ -1,7 +1,28 @@
 import { defineAuth } from '@aws-amplify/backend';
 
+/**
+ * Define and configure your auth resource
+ * When used alongside data, it is automatically configured as an auth provider for data
+ * @see https://docs.amplify.aws/gen2/build-a-backend/auth
+ */
 export const auth = defineAuth({
   loginWith: {
     email: true,
+    // add social providers
+    externalProviders: {
+      // first, create your secrets using `amplify sandbox secret`
```

Comment:
@see: https://docs.amplify.aws/gen2/deploy-and-host/sandbox-environments/features/#setting-secrets

Reply:
added